### ISSUE TITLE
Emoji's not displaying inside of mart, but are hoverable and do show tooltip.

### ISSUE THREAD

**Issue by danieltillero** [TypeCtrl/ngx-emoji-mart#339]
![Screenshot from 2020-06-24 10-27-28](https://user-images.githubusercontent.com/22438151/85579020-85435f00-b608-11ea-8191-7b3c5fbf5acf.png)

**Comment by danieltillero** [comment]
This hasn't always happened but please help me understand what is going on.

**Comment by Aleeexx** [comment]
Did you check in the browser's dev tools' network tab if the backgroundImage (containing the emojis) was loaded successfully? 
You can either load them from unpkg (default, https://www.npmjs.com/package/@ctrl/ngx-emoji-mart#sheet-sizes) or provide a sheet locally by yourself (by setting `backgroundImageFn`) https://www.npmjs.com/package/@ctrl/ngx-emoji-mart#components.

**Comment by Aleeexx** [comment]
Nope, no clue. I had the problem with emojis not being displayed when I provided a wrong sheet locally...

**Comment by scttcper** [comment]
I would check that other styles are not overriding those in emoji-mart

**Comment by Victorm0612** [comment]
I had the same problem. My solution was to add the **set** property to the emoji-mart component and use any option other than emojione. 

` 
          <emoji-mart
            class="emoji-mart"
            set="twitter"
            (emojiSelect)="addEmoji($event)"
            title="Choose your emoji…"></emoji-mart>
`

**Comment by saranglakare** [comment]
I faced the same problem today. It turns our `unpkg` is giving 502 randomly. Since no error is shown by the emoji picker, the user experience is bad. I am going to try and use `backgroundImageFn` to load it myself now. Let's see.
